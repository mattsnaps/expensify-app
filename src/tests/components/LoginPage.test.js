import React from "react";
import { shallow } from "enzyme";
import LoginPage from "../../components/LoginPage";

test('Login Page Render Test', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
});