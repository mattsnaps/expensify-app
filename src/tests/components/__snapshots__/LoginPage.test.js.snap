// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page Render Test 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Expensify App
    </h1>
    <p>
      The expense tracking app!
    </p>
    <button
      className="button"
      onClick="Boop"
    >
      Login with Google
    </button>
  </div>
</div>
`;
